docs/src/data/amplifyMock/components.tsx

```diff
@@ -0,0 +1,175 @@
+import { Flex, Icon, Button, Text, Divider } from '@aws-amplify/ui-react';
```

Comment:
Didn't we already add this?

Reply:
Maybe sync your remote branch (`avirajdh:poc-authenticator-demo`) with upstream (`aws-amplify:poc-authenticator-demo`)